scss/components/_button.scss

```diff
@@ -0,0 +1,77 @@
+@use "../common/forwarded" as helpers;
+
+.button {
+	position: relative;
+	display: block;
+	width: 170px;
+	height: 43px;
+	border-radius: 5px;
+	font-family: 'Montserrat', sans-serif;
+	font-weight: 600;
+	font-size: 10px;
+	line-height: 10px;
+	letter-spacing: 0.07em;
+	text-transform: uppercase;
+	color: helpers.$color-dark-orange;
+	background: helpers.$gradient-yellow;
+	box-shadow: 0px 4px 0px helpers.$color-light-orange;
+	@include helpers.zIndex(layer_middle);
```

Comment:
why do you need z-index here?

Reply:
Done.

Reply:
It must be redone in new refactoring branch